docs/src/faqs/setup_packaging.rst

```diff
@@ -0,0 +1,108 @@
+What do I need to know about using ``setup.py`` to package dependencies in Klio?
+================================================================================
+
+Using ``setup.py`` or a ``requirements.txt`` file to specify dependencies to install on workers was the original means by which a Python Beam job developer could customize the environment in which their job ran. You can read more about it in the `here <https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/>`_.
```

Comment:
I wouldn't include "or a `requirements.txt` file" since we don't exactly support that.

Reply:
Also, don't forget [semantic new lines](https://rhodesmill.org/brandon/2012/one-sentence-per-line/).